src/vm/i386/cgenx86.cpp

```diff
@@ -299,7 +299,12 @@ void TransitionFrame::UpdateRegDisplay(const PREGDISPLAY pRD)
 
     MethodDesc * pFunc = GetFunction();
     _ASSERTE(pFunc != NULL);
+
+#ifdef WIN64EXCEPTIONS
+    UpdateRegDisplayHelper(pRD, 0);
```

Comment:
@parjong I am also not sure I understand your reply here. The TransitionFrame::UpdateRegDisplay effectively unwinds to a managed method. So I was assuming that we would use the `UpdateRegDisplayHelper(pRD, pFunc->CbStackPop())` for WIN64EXCEPTIONS too so that pRD->pCurrentContext->Esp gets ESP before the stack arguments were pushed, like the UnwindStackFrame does.